release_build_files/readme.md

```diff
@@ -580,6 +580,9 @@ code.
         ([#745](https://github.com/firebase/firebase-cpp-sdk/pull/745))
     -   Messaging (Android): Fixed crash during initialization.
         ([#760](https://github.com/firebase/firebase-cpp-sdk/pull/760))
+    - Remote config (Desktop): Fixed cache expiration time value used by
```

Comment:
Can you fix the indents to match the above lines? (4 spaces before the hyphen then 3 spaces after the hyphen; on the following lines, 8 space indent, no tabs). Also could you fix the capitalization to "Remote Config"?

Thanks!